### PR TITLE
Fixed issue #1354

### PR DESCRIPTION
**Pull request by gary-boop**
Fixes for all points in issue #1354
Sections updated: 6.7.2, 6.7.3 and 6.7.4

### CODE REVIEW

**Review by pragyansharma24 — APPROVED** [review]

File: 06-process.Rmd [hunk 1]
```diff
@@ -320,6 +320,7 @@ The following chart, taken from [@ncbi] shows re-admission rates for different p
 * **Other important KPIs**: Some of the other important KPIs for healthcare providers include, patient satisfaction scores, asset utilization for operating theatres, diagnostic and other equipment and patient footfalls.
 
 **In the publich health sector, a few of the common data being generated are**:
```
Comment (pragyansharma24) [inline 06-process.Rmd]: line 322: Spelling of public
line 344:  **analyzing** the prevalence of **diseases**

**Review by pragyansharma24 — CHANGES_REQUESTED** [review]

File: 06-process.Rmd (hunk 1 above)
Comment (gauravhassija) [reply 06-process.Rmd]: Hey @pragyansharma24, the changes you mentioned are out of the scope of issue #1354 
I have worked on the parts that were highlighted in the issue and not the overall section.

Comment (mschermann) [reply 06-process.Rmd]: @gauravhassija Spelling should not be out of scope. Please fix and merge.

Comment (gauravhassija) [reply 06-process.Rmd]: Fixed the spelling errors

**Review by pragyansharma24 — APPROVED** [review]

**Pull request merged**